src/Init/Data/Json/Printer.lean

```diff
@@ -0,0 +1,75 @@
+/-
+Copyright (c) 2019 Gabriel Ebner. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Gabriel Ebner, Marc Huisinga
+-/
+prelude
+import Init.Lean.Data.Format 
+import Init.Data.Json.Basic
+
+open Lean
+
+namespace Json
+
+private def escapeAux (c : Char) (acc : String) : String :=
+-- escape ", \, \n and \r, keep all other characters ≥ 0x20 and render characters < 0x20 with \u
+if c = '"' then -- hack to prevent emacs from regarding the rest of the file as a string: "
```

Comment:
Is this still necessary?

Reply:
The trick to prevent Emacs from highlighting the rest of the file as a string? It is still necessary if you want to view the file in Emacs.

Reply:
Right, we cannot set `'` as both a word constituent and a string delimiter. Well, we probably could with custom code, but we may as well wait for server-side highlighting.